TombEngine/Objects/Generic/puzzles_keys.cpp

```diff
@@ -108,13 +122,26 @@ void PuzzleHoleCollision(short itemNumber, ItemInfo* laraItem, CollisionInfo* co
 		puzzleType = PuzzleType::Specfic;
 	}
 
-	if (((IsHeld(In::Action) || g_Gui.GetInventoryItemChosen() != NO_VALUE) &&
+	bool isUnderwater = (player.Control.WaterStatus == WaterStatus::Underwater);
+
```

Comment:
You can remove empty lines between these 3 bool definitions.

Reply:
Done